templates/socialaccount/snippets/social-buttons.html

```diff
@@ -23,10 +23,10 @@
   {% endif %}
   {% is_next_url_admin as show_superuser_login %}
   {% if show_superuser_login and auth_process == 'login' %}
-  <a href="{% url 'admin:login' %}">
-    <button class="btn social-admin">
+  <a href="{% url 'admin:login' %}" class="d-none">
```

Comment:
@youriwims why is this set to `display: none`?

Reply:
It was primarily for design review. I can revert it now.

Reply:
yeah revert it. the only way to login in to review apps is using super users which means that this button needs to be there for testing.

For future reference as well: For design review, if they want to see the pages without the django admin button, you just need to change the url in the `next` query string. For example: https://network-pulse-api-stagi-pr-505.herokuapp.com/accounts/login/?next=/api/pulse/v2/entries/